addons/l10n_ar_withholding/wizards/account_payment_register.py

```diff
@@ -23,32 +23,24 @@ def _compute_withholdings(self):
             rec.l10n_ar_withholding_ids = [Command.clear()] + [Command.create({'tax_id': x.id, 'base_amount': 0}) for x in taxes]
         (self - supplier_recs).l10n_ar_withholding_ids = False
 
-    # @api.depends('l10n_latam_check_id', 'l10n_ar_withholding_ids.amount')
-    # def _compute_amount(self):
-    #     super()._compute_amount()
-    #     for wizard in self.filtered(lambda x: x.l10n_ar_withholding_ids and x.l10n_latam_check_id):
-    #         wizard.currency_id = wizard.l10n_latam_check_id.currency_id
-    #         wizard.l10n_ar_net_amount = wizard.l10n_latam_check_id.amount
-    #         factor = wizard._get_net_amount_simulated_factor()
-    #         withholding_net_amount = min(wizard.l10n_latam_check_id.amount , wizard.net_amount)
-    #         wizard.amount = withholding_net_amount * factor + (wizard.l10n_latam_check_id.amount - withholding_net_amount)
+    def _compute_amount(self):
+        super()._compute_amount()
+        for wizard in self.filtered(lambda x: x.l10n_ar_withholding_ids and x.l10n_latam_check_id):
+            wizard.currency_id = wizard.l10n_latam_check_id.currency_id
+            wizard.l10n_ar_net_amount = wizard.l10n_latam_check_id.amount
```

Comment:
This should go in the other compute...  (hopefully there is a way to avoid to compute twice, but already one field is stored and the other not.

Reply:
DONE